tee-worker/client-api/sidechain-api/README.md

```diff
@@ -8,42 +8,48 @@ These types were auto generated using [Polkadot.js Type Generation](https://polk
 
 1. Install the package from NPM
 
-   ```typescript
-   npm install @litentry/sidechain-api
-   ```
+    ```typescript
+    npm install @litentry/sidechain-api
+    ```
 
 2. Import type definitions as needed:
 
-   ```typescript
-   import type { LitentryIdentity } from '@litentry/sidechain-api';
-   
-   function myFunction(identity: LitentryIdentity) {
-     // ...
-   }
-   ```
+    ```typescript
+    import type { LitentryIdentity } from "@litentry/sidechain-api";
+
+    function myFunction(identity: LitentryIdentity) {
+        // ...
+    }
+    ```
 
 ## Versions
 
 This package is distributed under two main tags: `next` and `latest`.
 
-Versions in the pattern of `x.x.x-next.x` feature the most recent code version to work with `tee-dev`. E.g., `1.0.0-next.0`. Once stable and once the Litentry Protocol is upgraded, the version will be tagged as `latest` and should be used against `tee-prod`. E.g.`1.0.0`. 
+Versions in the pattern of `x.x.x-next.x` feature the most recent code version to work with `tee-dev`. E.g., `1.0.0-next.0`. Once stable and once the Litentry Protocol is upgraded, the version will be tagged as `latest` and should be used against `tee-prod`. E.g.`1.0.0`.
 
 ## Publish new versions
 
+1. [Update your published package version number](https://docs.npmjs.com/updating-your-published-package-version-number)
+
 1. Build the package
 
-   ```typescript
-   pnpm run build
-   ```
+    ```s
+    pnpm run build
+    ```
+
+1. Publish the distribution files
 
-2. [Update your published package version number](https://docs.npmjs.com/updating-your-published-package-version-number)
+    Use `next` tag for preview versions. Use `latest` tag for
```

Comment:
Should be consistent with the parachain-api? Also, it seems that the description is missing?